components/annot-tab.vue

```diff
@@ -1,36 +1,23 @@
 <template>
 
-    <div>
+    <div :data-cy="activeCategory">
 
         <!-- Explanation text for this annotation tab -->
-        <annot-explanation :explanation="details.explanation" :index="details.id" :data-cy="'annot-expl-' + title" />
+        <annot-explanation :data-cy="('annot-expl-' + activeCategory)" />
 
         <!-- Lists all the columns linked to the category of this annotation tab -->
-        <annot-columns
-            :active-category="details.category"
```

Comment:
@surchs Currently, information needed for specific annotation tabs and their components are stored in `annotationDetails` in the store or pulled from the former computed properties of `annot-tab`. Presumably in the refactor, each child component knows how to get the information it needs from the store now based solely on the `activeCategory` passed to `annot-tab`. 

Sound correct?

Reply:
Yes, exactly. The components are now getting everything that is global state (most things with a few exceptions) directly from the store, either via getters or by directly reading state objects.